packages/compass-aggregations/src/components/pipeline-explain/explain-query-performance.tsx

```diff
@@ -40,14 +40,18 @@ export const ExplainQueryPerformance: React.FunctionComponent<ExplainQueryPerfor
       >
         <Subtitle>Query Performance Summary</Subtitle>
         <div className={statsStyles}>
-          <div className={statItemStyles}>
-            <Body>Documents returned:</Body>
-            <Body weight="medium">{nReturned}</Body>
-          </div>
-          <div className={statItemStyles}>
-            <Body>Actual query execution time(ms):</Body>
-            <Body weight="medium">{executionTimeMillis}</Body>
-          </div>
+          {nReturned > 0 && (
+            <div className={statItemStyles}>
+              <Body>Documents returned:</Body>
+              <Body weight="medium">{nReturned}</Body>
+            </div>
+          )}
```

Comment:
It is probably possible for `nReturned` to be zero if query did not match any documents, right? 🤔